src/styles/catalog.css

```diff
@@ -4,7 +4,7 @@
 .layout-row {
   display: flex;
   flex-flow: row wrap;
-  justify-content: flex-start;
+  justify-content: space-between;
```

Comment:
I think this change alters the layout of the "additional datasets" section when there are few results. Compare the [existing staging](https://planetarycomputer-staging.microsoft.com/catalog?tags=Water) to [this branch](https://icy-meadow-0fc35e30f-211.eastus2.azurestaticapps.net/catalog?tags=Water)

![image](https://user-images.githubusercontent.com/1014341/147996579-cee296d0-98f9-4e17-bacc-171b9c700433.png)

Reply:
Great catch.